tests/ssl-opt.sh

```diff
@@ -8837,6 +8837,36 @@ run_test    "TLS1.3: Test client hello msg work - openssl" \
             -c "<= parse certificate verify"          \
             -c "mbedtls_ssl_tls13_process_certificate_verify() returned 0"
 
+requires_openssl_tls1_3
```

Comment:
As mentioned offline, for testing server authentication based on RSA_PSS_RSAE_SHA256, look in ssl-opt.sh in the prototype branch (https://github.com/hannestschofenig/mbedtls) the tests with a header containing "Server authentication with RSA_PSS_RSAE_SHA256".

Reply:
Thanks a lot. Will check that.